docs/collecting-container-logs.md

```diff
@@ -57,6 +57,73 @@ This feature is enabled by default and the default regex will catch logs startin
 
 This feature can rarely cause problems by merging together lines which are supposed to be separate. In that case, feel free to disable it.
 
+### Log format
+
+There are two log formats available: `json` and `text`. `json` is the default.
+
+#### `json` log format
+
+Logs formatted as `json` are wrapped in a JSON object with additional properties, with the log body residing under the `log` key.
+
+For example, if we take the following log line:
+
+```text
+2007-03-01T13:00:00Z I am a log line
+```
+
+it will show up in Sumo Logic as:
+
+```javascript
+{
+  log: "2007-03-01T13:00:00Z I am a log line",
+  stream: "stdout",
+  timestamp: 1673627100045
+}
+```
+
+If the log line contains json to begin with:
+
+```json
+{"log_property": "value","text": "I am a json log"}
+```
+
+it will be displayed as a nested object inside the `log` key:
+
+```javascript
+{
+  log: {
+    log_property: "value",
+    text: "I am a json log"
+  },
+  stream: "stdout",
+  timestamp: 1673627100045
+}
+```
+
+#### `text` log format
+
+If you'd like to disable the wrapper, you can do so by setting:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      format: text
+```
+
+Then the log lines from the previous section will look like the following in Sumo Logic:
+
+```text
+2007-03-01T13:00:00Z I am a log line
+```
+
+```javascript
+{
+  log_property: "value",
+  text: "I am a json log"
+}
+```
```

Comment:
Could you add example without any property as well?

Reply:
Do you think that would be helpful? I've already got four examples, fitting in two more makes the whole section less legible in general.

Reply:
I rewrote this whole section while adding `json_merge`.